source/common/http/conn_manager_impl.h

```diff
@@ -354,11 +356,18 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     Tracing::SpanPtr active_span_;
     ResponseEncoder* response_encoder_{};
     Stats::TimespanPtr request_response_timespan_;
-    // Per-stream idle timeout.
+    // Per-stream idle timeout. This timer gets reset whenever activity occurs on the stream, and,
+    // when triggered, will close the stream.
     Event::TimerPtr stream_idle_timer_;
-    // Per-stream request timeout.
+    // Per-stream request timeout. This timer is enabled when the stream is created and disabled
+    // when the downstream closes the connection. If triggered, it will close the stream.
```

Comment:
I think there are some errors in this comment.  This seems to be a stream timeout, but there are references to downstream closing the connection.

Reply:
Thanks, fixed.